sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestMapBinder.java

```diff
@@ -141,7 +141,7 @@ public Result map(ResultSet r, StatementContext ctx)
         }
     }
 
-    static class Result
+    public static class Result
```

Comment:
I was surprised by this one. IIRC I got an illegal access exception when the class was non-public